src/config/Config.zig

```diff
@@ -4754,6 +4754,11 @@ pub const Keybinds = struct {
                 .{ .key = .{ .translated = .end }, .mods = .{ .super = true } },
                 .{ .scroll_to_bottom = {} },
             );
+            try self.set.put(
+                alloc,
+                .{ .key = .{ .translated = .j }, .mods = .{ .super = true } },
+                .{ .scroll_to_selection = {} },
+            );
```

Comment:
I don't think this needs a default binding. If you want one, we should discuss it. Why did you choose this?

Reply:
> Why did you choose this?

It’s the binding that Terminal.app uses for Edit -> Find -> Jump to Selection, but admittedly I actually never knew it existed until recently. I was experimenting with adding a Find bar to Ghostty and noticed that Jump to Selection was a thing